site/docs/install.md

```diff
@@ -5,14 +5,17 @@ title: Installing Bazel
 
 # Installing Bazel
 
-See the instructions for installing Bazel on:
+The Bazel team officially supports Bazel on the following platforms. See the instructions for installing Bazel on:
```

Comment:
The original sentence reads a bit strange to me now that it's not alone. Do we even need it now?

Reply:
Removed.